src/test/java/teammates/test/cases/ui/browsertests/AllJsTests.java

```diff
@@ -21,23 +21,23 @@
 public class AllJsTests extends BaseUiTestCase{
     
     private static Browser browser;
+    private static QUnitPage page;
+    private static final float MIN_COVERAGE_REQUIREMENT = 30;
     
     @BeforeClass
     public static void setUp() throws IOException {
         printTestClassHeader();
         browser = BrowserPool.getBrowser();
-        AppPage.getNewPageInstance(browser).navigateTo(createLocalUrl("/allJsUnitTests.html"));
+        page = AppPage.getNewPageInstance(browser)
+                      .navigateTo(createLocalUrl("/allJsUnitTests.html?coverage"))
+                      .changePageType(QUnitPage.class);
+        page.waitForPageToLoad();
     }
 
     @Test
     public void executeJsTests() {
-        String totalCasesXpathQuery = "//span[@class='total']",
-               failedCasesXpathQuery = "//span[@class='failed']";
-        
-        int totalCases = Integer.parseInt(browser.driver
-                .findElement(By.xpath(totalCasesXpathQuery)).getText());
-        int failedCases = Integer.parseInt(browser.driver
-                .findElement(By.xpath(failedCasesXpathQuery)).getText());
+        int totalCases = page.getTotalCases();
+        int failedCases = page.getFailedCases();
```

Comment:
@wkurniawan07 here: yup it is, if the number of failed cases is non-zero this test will fail.